packages/localizations/src/en-US.ts

```diff
@@ -354,6 +354,12 @@ export const enUS: LocalizationResource = {
         destructiveActionSubtitle: 'Remove this web3 wallet from your account',
         destructiveAction: 'Remove wallet',
       },
+      dangerSection: {
+        title: 'Danger',
+        deleteAccountButton: 'Delete Account',
+        deleteAccountTitle: 'Delete Account',
+        deleteAccountDescription: 'Delete your account and all its associated data',
+      },
```

Comment:
Just as a note, I only ran the english translation here. I can probably cover a couple more but I'm not sure how we normally tackle translation.

Reply:
😀 We officially support only the english translation. It's a personal decision if you want to do the rest as well.